### PR TITLE
HRIN-446: Added drush commands

### PR DESCRIPTION
**Pull request by martinyde**
https://jira.itkdev.dk/browse/HRIN-446

- Minor changes to config.
- Drush commands related to public meeting content type.
- Use of content_state in templates.

### CODE REVIEW

File: web/modules/custom/hoeringsportal_public_meeting/hoeringsportal_public_meeting.module [hunk 1]
```diff
@@ -12,7 +12,8 @@ use Drupal\node\NodeInterface;
  * Set last meeting time for public meeting, based on Pretix field time_to values.
  */
 function hoeringsportal_public_meeting_node_presave(NodeInterface $node) {
-  if ($node->bundle() == 'public_meeting') {
+  $user = Drupal::currentUser();
+  if ($user->id() > 0 && $node->bundle() == 'public_meeting') {
```
Comment (rimi-itk) [inline web/modules/custom/hoeringsportal_public_meeting/hoeringsportal_public_meeting.module]: I case you mean "user is authenticated” (which I suppose you do), using `Drupal::currentUser()->isAuthenticated()` will make the code easier to understand.

Use `===` for strict comparison.

Comment (martinyde) [reply web/modules/custom/hoeringsportal_public_meeting/hoeringsportal_public_meeting.module]: Fixed

File: web/modules/custom/hoeringsportal_content_blocks/templates/hoeringsportal-meeting-warning.html.twig [hunk 2]
```diff
@@ -0,0 +1,16 @@
+{#
+/**
+ * @file
+ * Display of hearing warning
+ *
+ * Available variables:
+ * - config: The available texts
+ */
+#}
+{% if (config.deadline_passed == 'finished') %}
```
[the remark this answers is not in the document]
Comment (martinyde) [reply web/modules/custom/hoeringsportal_content_blocks/templates/hoeringsportal-meeting-warning.html.twig]: Fixed

File: web/modules/custom/hoeringsportal_content_blocks/src/Plugin/Block/MeetingWarning.php [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+ * )
+ */
+class MeetingWarning extends BlockBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function build() {
+    // Get current node.
+    $node = \Drupal::routeMatch()->getParameter('node');
```
[the remark this answers is not in the document]
Comment (martinyde) [reply web/modules/custom/hoeringsportal_content_blocks/src/Plugin/Block/MeetingWarning.php]: Changed

File: web/modules/custom/hoeringsportal_content_blocks/src/Plugin/Block/MeetingWarning.php [hunk 4]
```diff
@@ -12,16 +15,61 @@
...
+   * @param string $plugin_id
+   *   Block plugin id.
+   * @param mixed $plugin_definition
+   *   Block plugin definition.
+   * @param \Drupal\Core\Routing\CurrentRouteMatch $routeMatch
+   *   The route match.
+   */
+  public function __construct(array $configuration, $plugin_id, $plugin_definition, CurrentRouteMatch $routeMatch) {
+    parent::__construct($configuration, $plugin_id, $plugin_definition);
+    $this->currentNode = $routeMatch->getParameter('node');
```
Comment (rimi-itk) [inline web/modules/custom/hoeringsportal_content_blocks/src/Plugin/Block/MeetingWarning.php]: You should not get the current node here, but store the service for later use, e.g.
```php
    $this->routeMatch = $routeMatch;
```

File: web/modules/custom/hoeringsportal_content_blocks/src/Plugin/Block/MeetingWarning.php [hunk 5]
```diff
@@ -12,16 +15,61 @@
...
 
   /**
    * {@inheritdoc}
    */
   public function build() {
-    // Get current node.
-    $node = \Drupal::routeMatch()->getParameter('node');
-    if (isset($node) && $node->field_content_state->value) {
-      $config['deadline_passed'] = $node->field_content_state->value;
+    if (isset($this->currentNode) && $this->currentNode->field_content_state->value) {
```
Comment (rimi-itk) [inline web/modules/custom/hoeringsportal_content_blocks/src/Plugin/Block/MeetingWarning.php]: You should get the current from the routeMatch service here, e.g. `$node = $this->routeMatch->getParameter('node')`.

**Pull request merged**